xonsh/prompt_toolkit_shell.py

```diff
@@ -0,0 +1,113 @@
+"""The prompt_toolkit based xonsh shell"""
+import os
+import builtins
+from warnings import warn
+
+from prompt_toolkit.shortcuts import create_cli, create_eventloop
+from pygments.token import Token
+
+from xonsh.base_shell import BaseShell
+from xonsh.history import LimitedFileHistory
+from xonsh.pyghooks import XonshLexer
+from xonsh.tools import format_prompt_for_prompt_toolkit
+from xonsh.prompt_toolkit_completer import PromptToolkitCompleter
+
+from prompt_toolkit.key_binding.manager import KeyBindingManager
```

Comment:
PEP8 please move this import up to be next to the shortcuts importer above. Thanks!